assets/quick-order-list.js

```diff
@@ -67,16 +66,15 @@ class QuickOrderList extends HTMLElement {
       add: 'ADD',
       update: 'UPDATE'
     }
+    this.allInputs = this.querySelectorAll('input[type="number"]');
+    this.allInputs.forEach((input) => {input.addEventListener('focus', this.switchVarints.bind(this))});
```

Comment:
1. I believe we do scope it to the quick order list only. We use `this` and not `document` which means we scope exactly within our custom <quick-order-list> element. Am I right or missing something? 🤔

2. That makes sense to me 👍. I also changed event listener from 'focus' to 'focusin' because 'focus' cannot be bubbled.

Reply:
(1) Ah yes, sorry, I missed that 😅
(2) TIL about focus not bubbling! Thanks for finding a good alternative 👍 but fine to switch it back if focusin affects the UX negatively.